CODEOWNERS

```diff
@@ -7,38 +7,40 @@
 
 # List of components with at least two owners as specified in docs/crate-ownership.
 
+components/calendar/              @Manishearth @sffc
 components/collator/              @hsivonen @echeran
+components/collections/           @echeran @iainireland
 components/datetime/              @zbraniecki @gregtatum @nordzilla
 components/decimal/               @sffc
-components/icu/                   @unicode-org/icu4x-owners
-components/icu4x/                 @unicode-org/icu4x-owners
-components/locale_canonicalizer/  @dminor @zbraniecki
+components/list/                  @robertbastian
 components/locid/                 @zbraniecki @nciric
+components/locid_transform/       @dminor @zbraniecki
 components/normalizer/            @hsivonen @echeran
 components/plurals/               @zbraniecki @sffc
-components/uniset/                @echeran @iainireland
+components/timezone/              @nordzilla
 experimental/bies/                @sffc
-experimental/calendar/            @Manishearth @sffc
-experimental/codepointtrie/       @echeran
-experimental/provider_ppucd/      @echeran
+experimental/compactdecimal/      @eggrobin
+experimental/displaynames/        @sffc
+experimental/relativetime/        @pdogr
 experimental/segmenter/           @aethanyc @makotokato @sffc
 ffi/capi/                         @Manishearth
-ffi/cpp/                          @Manishearth
```

Comment:
should probably add a blanket ffi/ instead

Reply:
Everything that's not listed defaults to `icu4x-owners`, so I think it's fine?

Reply:
Mostly to say who the explicit point of contact is